### PR TITLE
Write CSV files to existing Google spreadsheets.

### PR DESCRIPTION
**Pull request by rickmcphee**
Create a small helper class to allow a CSV file to be uploaded to an
existing Google spreadsheet.

### CODE REVIEW

File: requirements-to-freeze.txt [hunk 1]
```diff
@@ -1,4 +1 @@
-boto3
```
Comment (bsiemon) [inline requirements-to-freeze.txt]: Why remove these?

Comment (rickmcphee) [inline requirements-to-freeze.txt]: These are already frozen in requirements.txt, so just deleted them and added the new dependency.

Comment (bsiemon) [inline requirements-to-freeze.txt]: The idea with this file is that it is the immediate dependencies of the project which are then frozen along with whatever transitive dependencies exist.

Comment (rickmcphee) [inline requirements-to-freeze.txt]: So, the requirements-to-freeze.txt file is where the developer maintains the packages that this one imports?  Then freezing into requirements.txt describes those plus the transitive dependencies?

Comment (rickmcphee) [inline requirements-to-freeze.txt]: Added back the erroneously removed packages from `requirements-to-freeze.txt`.

**Comment by bsiemon** [comment]
@Jan has a good overview of how these requirements files work. He set it up and I liked it but not sure if I explained it well.

**Comment by rickmcphee** [comment]
I chatted with @Jan when you raised these questions, and I understand it now.  I added the direct dependencies back into the `requirements-to-freeze.txt` file.

**Pull request merged**